### PR TITLE
#3551 - macro oxygen cannot be a leaving group in a chemical reaction

### PR DESCRIPTION
**Pull request by ivan-arkhipov-epam**
## How the feature works? / How did you fix the issue?

Value of O (oxygen) is displayed as OH. The rest is displayed as is.

image

Most of changes in this PR are not related to the issue, I just had to make them because exporting Peptide from ketcher-core affected surprising amount of tests.

## Check list
- [x] unit-tests written
- [ ] e2e-tests written
- [ ] documentation updated
- [x] PR name follows the pattern `#1234 – issue name`
- [x] branch name doesn't contain '#'
- [x] PR is linked with the issue
- [x] base branch (master or release/xx) is correct
- [x] task status changed to "Code review"
- [ ] reviewers are notified about the pull request

### CODE REVIEW

File: packages/ketcher-polymer-editor-react/src/components/modal/monomerConnection/MonomerConnections.tsx [hunk 1]
```diff
@@ -171,6 +173,16 @@ function AttachmentPointSelectionPanel({
       return acc;
     }, {});
 
+  const getLeavingGroupValue = (attachmentPoint): LeavingGroupValue => {
+    if (!monomerLeavingGroups) {
+      return 'H';
+    }
+    const leavingGroupValue = monomerLeavingGroups[attachmentPoint];
```
Comment (ivan-arkhipov-epam) [inline packages/ketcher-polymer-editor-react/src/components/modal/monomerConnection/MonomerConnections.tsx:180]: I'm not sure about naming here. If anyone could suggest a better term, I would appreciate.

File: packages/ketcher-polymer-editor-react/src/components/modal/monomerConnection/MonomerConnections.tsx [hunk 2]
```diff
@@ -19,6 +19,8 @@ import {
 } from './styledComponents';
 import { MonomerConnectionProps } from '../modalContainer/types';
 
+type LeavingGroupValue = 'O' | 'OH' | 'H';
```
Comment (ivan-arkhipov-epam) [inline packages/ketcher-polymer-editor-react/src/components/modal/monomerConnection/MonomerConnections.tsx:22]: If anyone has a feeling that this declaration would look better in another file (some place where all typings live?) please let me know. I just don't know where I should put it.

File: packages/ketcher-polymer-editor-react/src/components/modal/monomerConnection/MonomerConnections.tsx [hunk 3]
```diff
@@ -19,6 +19,8 @@ import {
 } from './styledComponents';
 import { MonomerConnectionProps } from '../modalContainer/types';
 
+type MonomerCap = 'O' | 'OH' | 'H';
```
Comment (ivan-arkhipov-epam) [inline packages/ketcher-polymer-editor-react/src/components/modal/monomerConnection/MonomerConnections.tsx:22]: If anyone has a feeling that this declaration would look better in another file (some place where all typings live?) please let me know. I just don't know where I should put it.

File: packages/ketcher-polymer-editor-react/src/components/modal/monomerConnection/MonomerConnections.tsx [hunk 4]
```diff
@@ -159,6 +161,15 @@ function AttachmentPointSelectionPanel({
     [bonds],
   );
 
+  const getMonomerCap = (attachmentPoint): MonomerCap => {
```
Comment (ivan-arkhipov-epam) [inline packages/ketcher-polymer-editor-react/src/components/modal/monomerConnection/MonomerConnections.tsx:164]: I'm not sure about naming here. If anyone could suggest a better term, I would appreciate.

**Review by OlgaMazurina — APPROVED** [review]

File: packages/ketcher-polymer-editor-react/src/components/modal/monomerConnection/MonomerConnections.tsx (hunk 3 above)
Comment (rrodionov91) [reply packages/ketcher-polymer-editor-react/src/components/modal/monomerConnection/MonomerConnections.tsx:22]: I would suggest to rename it to 'LeavingGroup'

File: packages/ketcher-polymer-editor-react/src/components/modal/monomerConnection/MonomerConnections.tsx (hunk 4 above)
Comment (rrodionov91) [reply packages/ketcher-polymer-editor-react/src/components/modal/monomerConnection/MonomerConnections.tsx:164]: I would suggest to rename it to 'getLeavingGroup'

**Pull request merged**